src/shared/util.js

```diff
@@ -702,6 +702,10 @@ class FeatureTest {
     return shadow(this, "isEvalSupported", isEvalSupported());
   }
 
+  static get isPath2DSupported() {
+    return shadow(this, "isPath2DSupported", typeof Path2D !== "undefined");
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/API/Path2D#browser_compatibility all supported browsers natively support `Path2D`, so I assume this is only necessary for NodeJS? If so, would it make sense to add a comment to note that since this is otherwise a widely available web API (in contrast to the other feature tests here)?

Reply:
Good point; maybe we should just check `!isNode` instead to make this clearer?

Reply:
That would also work indeed.